src/gauge.js

```diff
@@ -131,11 +131,30 @@
       return percentage * gaugeSpanAngle / 100;
     }
 
-    function normalize(value, min, limit) {
-      var val = Number(value);
-      if(val > limit) return limit;
-      if(val < min) return min;
-      return val;
+    /**
+     * Check value. If value is number - set it to array
+     * @param {Array/Number} value
+     * @param {Number} min
+     * @param {Number} limit
+     * @param {String} color
+     * @return {Array}
+     */
+    function normalizeMultipleValue(value, min, limit, color) {
+      // If value is single (number) - set it ro array
+      // Value should be in format [{value: number, color: 'string'}]
+      if (!Array.isArray(value)) value = [{value, color}];
+
+      value.map(function(v) {
+        var currentValue = Number(v.value);
+        if(currentValue > limit) currentValue = limit;
+        if(currentValue < min) currentValue = min;
+        v.value = currentValue;
+        return v;
+      });
+
+      return value.sort(function(a, b) {
+        return parseFloat(b.value) - parseFloat(a.value);
```

Comment:
value already a number, parseFloat is useless

Reply:
Done